src/client/components/LocationComponent/LocationComponent.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class LocationComponent extends React.Component {
```

Comment:
Did you learn both functional and class based React? Probably best if we decide on one or the other

Reply:
Hi, thank you for the feedback. We did learn both, which one should we use?

Reply:
Let's stick with the functional approach then